app/src/main/java/ru/vssemikoz/newsfeed/MainActivity.java

```diff
@@ -26,6 +27,8 @@
 import ru.vssemikoz.newsfeed.models.NewsItem;
 
 public class MainActivity extends AppCompatActivity implements PickCategoryDialog.OnCategorySelectedListener {
+    String TAG = "MyLog";
+    boolean favoriteNewsState = false;
     Category category = Category.ALL;
     String KEY;
     MainApplication mainApplication;
```

Comment:
Do you remember our talk about visibility modifiers? Which modifiers should be used for class members?